app/apps/server/communication/rest.js

```diff
@@ -110,10 +112,14 @@ export class AppsRestApi {
 						return API.v1.failure({ error: 'Unauthorized' });
 					}
 
+					const subscribeRoute = this.queryParams.details === 'true' ? 'subscribe/details' : 'subscribe';
+
+					const seats = Users.findActive().count() - Users.findActiveRemote().count();
```

Comment:
I'm thinking on moving this to the Users model as `Users.getActiveLocalUserCount()` because it's being called in multiple places.

@geekgonecrazy what do you think?

Reply:
Yes!  That would likely be better then duplicating this logic 👍